Dockerfile

```diff
@@ -1,13 +1,26 @@
 FROM centos:7
+ARG JAVA_VERSION=1.8.0
 
-RUN yum -y install java-1.8.0-openjdk-devel && yum clean all
+RUN yum -y update \
+    && yum -y install java-${JAVA_VERSION}-openjdk-headless openssl \
+    && yum -y clean all
+
+# Set JAVA_HOME env var
 ENV JAVA_HOME /usr/lib/jvm/java
 
-ADD target/kafka-bridge.jar /
+# Add strimzi user with UID 1001
+# The user is in the group 0 to have access to the mounted volumes and storage
+RUN useradd -r -m -u 1001 -g 0 strimzi
+
+ARG strimzi_version=1.0-SNAPSHOT
```

Comment:
is this the version of the bridge? it should be 0.12.0-SNAPSHOT right now.

Reply:
This is just a default. So that does not matter what is there since it will be overwritten in the build. But is had a wrong name (should be `strimzi_kafka_bridge_version`).